libbeat/publisher/publish.go

```diff
@@ -79,6 +79,10 @@ type ShipperConfig struct {
 	Topology_expire       int
 	Tags                  []string
 	Geoip                 common.Geoip
+
+	// internal publisher queue sizes
+	QueueSize     *int `yaml:"queue_size"`
```

Comment:
I'm +1 on making this also visible in the config file but with a warning, that people that change it must understand what they are doing.